### PR TITLE
[CSE-50] Add support for exporting SHEQSY data

### PR DESCRIPTION
**Pull request by BCook98**
This setups an initial capability to export data from SHEQSY, the data sets are limited to the available APIs from SHEQSY (https://developers.sheqsy.com). Once they extend these it should be trivial to extend the data sets. 

When running the tool, at least one API token is required for either SafetyCulture or SHEQSY. If neither is provided or one fails to authenticate the tool will bail out.

### CODE REVIEW

File: cmd/safetyculture-exporter/cmd/root.go [hunk 1]
```diff
@@ -90,6 +90,10 @@ func configFlags() {
 	// TODO - Can we validate these tokens and throw error if they are wrong?
 	connectionFlags.StringP("access-token", "t", "", "API Access Token")
 	connectionFlags.String("api-url", "https://api.safetyculture.io", "API URL")
+	connectionFlags.StringP("sheqsy-username", "", "", "SHEQSY API Username")
+	connectionFlags.StringP("sheqsy-password", "", "", "SHEQSY API Password")
```
Comment (hkf57) [inline cmd/safetyculture-exporter/cmd/root.go:94]: I'm very on the edge about flat out declining this PR solely based on the fact that we have to use username/password auth.

**Review by hkf57 — COMMENTED** [review]
Overall LGTM but I have strong concerns around auth.

File: cmd/safetyculture-exporter/cmd/root.go (hunk 1 above)
Comment (BCook98) [reply cmd/safetyculture-exporter/cmd/root.go:94]: From my testing I don't think this is actually the username and password of an organisation. These are actually an key/secret pair for the company in SHEQSY.

Using my username and password did not work.

**Review by HarryTranAU — DISMISSED** [review]
awesome stuff 👍

**Review by HarryTranAU — APPROVED** [review]

**Review by hkf57 — APPROVED** [review]
I'd be interested to know who the target users are as well :)

**Pull request merged**